src/os/bluestore/BitMapAllocator.cc

```diff
@@ -22,11 +22,11 @@ BitMapAllocator::BitMapAllocator(CephContext* cct, int64_t device_size,
 				 int64_t block_size)
   : cct(cct)
 {
-  assert(ISP2(block_size));
   if (!ISP2(block_size)) {
     derr << __func__ << " block_size " << block_size
          << " not power of 2 aligned!"
          << dendl;
+    assert(false);
```

Comment:
can we something descriptive in assert also? just nit, to grep from any asserts in the logs.